### PR TITLE
ci(release-drafter): add release-drafter template for Plugins

### PR DESCRIPTION
**Pull request by h1dden-da3m0n**
**Description**

As discussed in jellyfin/jellyfin-plugin-ldapauth#66 here a proposal for a Release Drafter template.

**Change**

* add Release Drafter template config
* add Release Drafter readme (heavily inspired by the one found in [jenkinsci/.github](https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc))

**Issues**

* N/A

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      - test
+      - tests
+exclude-labels:
+  - no-changelog
+  - skip-changelog
+  - invalid
+
+template: |
+  <!-- Optional: add a release summary here -->
+  [Plugin build can be downloaded here](https://repo.jellyfin.org/releases/plugin/ldap-authentication/ldap-authentication_$NEXT_MAJOR_VERSION.0.0.0.zip).
```
Comment (oddstr13) [inline .github/release-drafter.yml:56]:
```suggestion
```
Not sure of how we'd further template that, better to just remove it I think?

File: .github/release-drafter.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+==== Global Configuration Notes
+
+There are some considerations about the default configuration:
+
+* Jellyfin plugins can use different versioning formats.
+  Release Drafter defaults to link:https://semver.org/[semver], but the majority of Jellyfin plugins use only the major version number.
+  We use it as a default in the global configuration, but it can be overridden (e.g. `version-template: $MAJOR.$MINOR.$PATCH`)
+* Next version number will be used by default as a next release name.
+  Another naming template can be defined by a `name-template` property, e.g. `name-template: My Plugin v$NEXT_PATCH_VERSION`
```
Comment (oddstr13) [inline .github/release-drafter.adoc:50]: `tag-template` should probably also be mentioned here?

File: .github/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  $CHANGES
```
Comment (oddstr13) [inline .github/release-drafter.yml:57]: Missing trailing newline

File: .github/release-drafter.yml (hunk 1 above)
Comment (h1dden-da3m0n) [reply .github/release-drafter.yml:56]: Unfortunately you are right, I seem to have forgotten about this yesterday (ups, sorry).

File: .github/release-drafter.adoc (hunk 2 above)
Comment (h1dden-da3m0n) [reply .github/release-drafter.adoc]: I hope my basic text is sufficient

File: .github/release-drafter.yml (hunk 3 above)
Comment (h1dden-da3m0n) [reply .github/release-drafter.yml]: Okay for some reason that final new line is present in my local tree yet missing in the remote :eyes:

Comment (h1dden-da3m0n) [reply .github/release-drafter.yml]: okay fixed, had to add two empty line breaks locally :thinking: :man_shrugging:

**Comment by h1dden-da3m0n** [comment]
sorry for the pin @oddstr13, but could u give this one a look when u have time?

**Comment by h1dden-da3m0n** [comment]
okay sorry for reviving this after such a long time, however two things:
1. regarding the outdated git sub-modules we have 2 options
  a. delete them and treat this meta repo just like the other [jellyfin-meta](https://github.com/jellyfin/jellyfin-meta) repo
  b. use dependabot to keep them up to date (personally I would favor a.)
2. my latest commit adds a `category-template` resulting in a slight change who change-logs would look like. e.g. based on the default template:
```markdown
## :sparkles: What's New?
### :tada: New features and improvements
* ...
...
```

**Review by oddstr13 — APPROVED** [review]
Looks like this one slipped thru the cracks, apologies.

I read thru both files, and I didn't spot anything obviously wrong.

**Comment by h1dden-da3m0n** [comment]
> Looks like this one slipped thru the cracks, apologies.
> 
> I read thru both files, and I didn't spot anything obviously wrong.

No worries, I also kind of had it on the back-burner as I was working on other things :wink:

**Review by oddstr13 — APPROVED** [review]

File: .github/release-drafter.adoc [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+=== Enabling Release Drafter GitHub Action in a Repository
+
+Have a look to the sample configuration available in the link:https://github.com/jellyfin/jellyfin-plugin-ldapauth/blob/master/.github/workflows/update-release-draft.yml[LDAP-Auth Plugin].
+
+=== Configuring Release Drafter
+
+After enabling RD, it needs to be configured in `.github/release-drafter.yml` in the master branch of your repository.
+The Jellyfin project provides a link:./release-drafter.yml[Global Configuration file], so a minimal configuration looks like this one:
+
+[source,yml]
```
Comment (crobibero) [inline .github/release-drafter.adoc:27]:
```suggestion
[source.yml]
```

Should this link to a file?

Comment (h1dden-da3m0n) [reply .github/release-drafter.adoc:27]: Sorry for the confusion, but no this is [actual syntax in `adoc`](https://docs.asciidoctor.org/asciidoctor/latest/syntax-highlighting/) and is the equivalent to

```markdown

```yaml 
yaml: true
\```

```

had to put that `\` there just to get the GitHub Markdown parser to not freak out for nested source listings xD

**Review by crobibero — APPROVED** [review]
Nothing stands out to me

**Review by joshuaboniface — APPROVED** [review]

**Comment by h1dden-da3m0n** [comment]
yay, thy for the merge :tada:

**Pull request merged**